src/Appwrite/Resque/Worker.php

```diff
@@ -26,4 +33,107 @@ public function tearDown(): void
     {
         $this->shutdown();
     }
+    /**
+     * Get internal project database
+     * @param string $projectId
+     * @return Database
+     */
+    protected function getInternalDB(string $projectId): Database
+    {
+        global $register;
+
+        $attempts = 0;
+        $max = 10;
+        $sleep = 2;
```

Comment:
I think we can move this vars to become class constants. We might also be able to create more code reuse here in the future, not urgent for this scope.